src/cosmos/Swap.fixture.tsx

```diff
@@ -68,13 +75,17 @@ function Fixture() {
       defaultOutputTokenAddress={defaultOutputToken}
       defaultOutputAmount={defaultOutputAmount}
       locale={locale}
-      jsonRpcEndpoint={jsonRpcEndpoint}
+      jsonRpcUrlMap={jsonRpcUrlMap}
+      defaultChainId={defaultChainId}
       provider={connector}
       theme={theme}
       tokenList={tokenList}
       width={width}
       routerUrl={routerUrl}
-      onConnectWallet={() => console.log('onConnectWallet')} // this handler is included as a test of functionality, but only logs
+      onConnectWalletClick={() => {
+        // e?.preventDefault()
```

Comment:
rm

Reply:
I think leaving this in as an example is fine, but I'd add a comment about the expected behavior if `e.preventDefault` *is* called.